plugins/ksp/testData/api/platformDeclaration.kt

```diff
@@ -0,0 +1,44 @@
+// WITH_RUNTIME
+// TEST PROCESSOR: PlatformDeclarationProcessor
+// EXPECTED:
+// Actual.kt : Clazz : true : false : Expect.kt
+// Actual.kt : Clazz.foo : true : false : Expect.kt
+// Actual.kt : Klass : true : false : Expect.kt
+// Actual.kt : bar : true : false : Expect.kt
+// Actual.kt : baz : true : false : Expect.kt
+// Coffee.java : Coffee : false : false
+// Coffee.java : Coffee.baz : false : false
+// Coffee.java : Coffee.foo : false : false
+// Expect.kt : Clazz : false : true : Actual.kt
+// Expect.kt : Clazz.foo : false : false
+// Expect.kt : Klass : false : true : Actual.kt
+// Expect.kt : bar : false : true : Actual.kt
+// Expect.kt : baz : false : true : Actual.kt
```

Comment:
Minor: It would be nice if you can show an empty list here so the expected result is more readable.